### ISSUE TITLE
Project still alive?

### ISSUE THREAD

**Issue by doodirock** [ed-data/fafsa-api#12]
So i noticed the API can't directly submit, but how much of the work load can it actually handle?

**Comment by kinlane** [comment]
Yes, this was a prototype I created on request from the White House to stimulate a series of conversations we were having with Dept of Ed. It jumpstarted 3 rounds of talks, but I have seen no other action. Honestly, it is something the industry should come together and to....and push Ed to accept the results.

**Comment by wingrunr21** [comment]
Is there an API now in production given the new mobile app that just launched?

**Comment by kinlane** [comment]
No that I know of -- here is where to tune in - https://github.com/usedgov/API-Program/issues/1
